extensions/src/test/java/org/pentaho/platform/web/http/api/resources/UserSettingsResourceTest.java

```diff
@@ -79,10 +79,10 @@ public void testGetUserSettings() throws Exception {
     when( userSettingService.getUserSettings() ).thenReturn( userSettings );
 
     when( userSettingsResource.getUserSettings() ).thenCallRealMethod();
-    ArrayList<Setting> response = userSettingsResource.getUserSettings();
+    SettingsWrapper response = userSettingsResource.getUserSettings();
     assertNotNull( response );
-    assertTrue( !response.isEmpty() );
-    assertTrue( response.get( 0 ).getName().equals( USER_SETTING_NAME ) );
-    assertTrue( response.get( 0 ).getValue().equals( USER_SETTING_VALUE ) );
+    assertTrue( !response.getSettings().isEmpty() );
```

Comment:
These "assertTrue" could be replaced with assertFalse and assertEquals... but I think this is not a priority (and there are plenty more of these cases everywhere).